Snippets/Snippets_3/Encryption/Configuration/encryption.config.xml

```diff
@@ -1,15 +1,16 @@
 ﻿<?xml version="1.0" encoding="utf-8" ?>
 <configuration>
-  <!-- startcode EncryptionFromAppConfig-->
+  <!-- startcode EncryptionFromAppConfig -->
   <configSections>
-    <section name="RijndaelEncryptionServiceConfig" 
-             type="NServiceBus.Config.RijndaelEncryptionServiceConfig, NServiceBus.Core"/>
+    <section
+      name="RijndaelEncryptionServiceConfig"
+      type="NServiceBus.Config.RijndaelEncryptionServiceConfig, NServiceBus.Core"/>
   </configSections>
-  <RijndaelEncryptionServiceConfig Key="gdDbqRpQdRbTs3mhdZh9qCaDaxJXl+e6">
+  <RijndaelEncryptionServiceConfig Key="gdDbqRpQdRbTs3mhdZh9qCaDaxJXl+e6" KeyIdentifier="B">
     <ExpiredKeys>
-      <add Key="abDbqRpQdRbTs3mhdZh9qCaDaxJXl+e6" />
+      <add Key="abDbqRpQdRbTs3mhdZh9qCaDaxJXl+e6" KeyIdentifier=";A" />
```

Comment:
The semi-colon is used to specify multiple key identifiers for a single key.

In this example the semi-colon results in a 0 length key identifier so it will be used when decrypting data where no key identifier is set as mentioned in 'Using the same key with and without a key identifier'